README.md

```diff
@@ -1,2 +1,12 @@
 # Suzaku starter
 A starter project for using the Suzaku framework.
+
+1. `git clone https://github.com/suzaku-io/suzaku.git`
+1. `cd suzaku`
+1. `sbt publishLocal`
+1. `cd ..`
+1. `git clone https://github.com/suzaku-io/suzaku-starter.git`
+1. `cd suzaku-starter`
+1. `sbt`
+1. \> `re-start`
+1. Open browser on `http://0.0.0.0:12121`. 
```

Comment:
This should be `localhost:12121` since the numerical IP address won't work.

Reply:
It is http://0.0.0.0:12121 now.

Reply:
That doesn't work as 0.0.0.0 is not a valid address. You need `localhost` instead :)

Reply:
On a Mac http://0.0.0.0:12121 works without a problem. If it is a bug, i would consider  this output as misleading.

`server Starting starter.server.WebServer.main()
[success] Total time: 1 s, completed Sep 6, 2017 7:54:55 AM
> server Server online at http://0.0.0.0:12121
`
I think its best to leave it up to you to provide a decent starter guide.

Reply:
At least on Windows the `0.0.0.0` doesn't work, and `localhost` is the norm.

> In the Internet Protocol Version 4, the address 0.0.0.0 is a non-routable meta-address used to designate an invalid, unknown or non-applicable target.

Reply:
Maybe it would be best to change the server config to use only `localhost` binding, to avoid this problem in the first place. It's not like users expect their local dev server to be open to internet anyway :)